grid-client/deployer/deployment_deployer_test.go

```diff
@@ -660,5 +656,131 @@ func TestDeploymentDeployer(t *testing.T) {
 		assert.Empty(t, dl.QSFS)
 		assert.Empty(t, dl.Zdbs)
 	})
+}
+
+func ExampleDeploymentDeployer_Deploy() {
+	const mnemonic = "<mnemonics goes here>"
+	const network = "<dev, test, qa, main>"
+
+	tfPluginClient, err := NewTFPluginClient(mnemonic, "sr25519", network, "", "", "", 0, false)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+
+	n := workloads.ZNet{
+		Name:        "network",
+		Description: "network for testing",
+		Nodes:       []uint32{nodeID},
+		IPRange: gridtypes.NewIPNet(net.IPNet{
+			IP:   net.IPv4(10, 1, 0, 0),
+			Mask: net.CIDRMask(16, 32),
+		}),
+		AddWGAccess: false,
+	}
+
+	vm := workloads.VM{
+		Name:        "vm",
+		Flist:       "https://hub.grid.tf/tf-official-apps/base:latest.flist",
+		CPU:         2,
+		Planetary:   true,
+		Memory:      1024,
+		Entrypoint:  "/sbin/zinit init",
+		EnvVars:     map[string]string{"SSH_KEY": "<ssh key goes here>"},
+		NetworkName: n.Name,
+	}
+
+	err = tfPluginClient.NetworkDeployer.Deploy(context.Background(), &n)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+
+	dl := workloads.NewDeployment("vm", nodeID, "", nil, n.Name, nil, nil, []workloads.VM{vm}, nil)
```

Comment:
Change "vm" to other name to prevent confusion (deployment and VMs can have different names)
Check batch example